### PR TITLE
Less tidy up

### PR DESCRIPTION
**Pull request by guywillis**
Resolves https://github.com/cgkineo/adapt-list/issues/15

### CODE REVIEW

File: less/list.less [hunk 1]
```diff
@@ -44,14 +41,11 @@
       width: 15%;
     }
   }
-  // --------------------------------------------------
 }
 
-// --------------------------------------------------
 // --------------------------------------------------
```
Comment (tomgreenfield) [inline less/list.less:46]: Should this line need to be removed? Not sure of your rules.

File: less/list.less [hunk 2]
```diff
@@ -98,35 +89,28 @@
...
+    .transition(all .8s ease-in-out);
+  }
 
-    .list__container .list__item.is-animating {
-      right: 0;
-      opacity: 1;
-    }
+  &.is-animated-list &__item.is-animating {
+    right: 0;
+    opacity: 1;
```
Comment (tomgreenfield) [inline less/list.less:108]: Please move this block to line 104 with a selector of `&.is-animating`.

File: less/list.less (hunk 1 above)
Comment (guywillis) [reply less/list.less:46]: The general comment rule of thumb is

```
// Comment
// --------------------------------------------------
```

```
// --------------------------------------------------
// Divider
// --------------------------------------------------
```

File: less/list.less (hunk 2 above)
Comment (guywillis) [reply less/list.less]: Done

**Review by tomgreenfield — APPROVED** [review]
👁️

**Review by kirsty-hames — APPROVED** [review]
👀

**Pull request merged**